packages/siop-oid4vp/lib/authorization-request/URI.ts

```diff
@@ -163,8 +164,9 @@ export class URI implements AuthorizationRequestURI {
     const requestObjectPayload: RequestObjectPayload = requestObjectJwt ? (parseJWT(requestObjectJwt).payload as RequestObjectPayload) : undefined
 
     if (requestObjectPayload) {
-      // Only used to validate if the request object contains presentation definition(s)
+      // Only used to validate if the request object contains presentation definition(s) | a dcql query
       await PresentationExchange.findValidPresentationDefinitions({ ...authorizationRequestPayload, ...requestObjectPayload })
+      await findValidDcqlQuery({ ...authorizationRequestPayload, ...requestObjectPayload })
```

Comment:
What is the reason for the static import change. Not like this is easier to read or anything

Reply:
Not my code again: https://github.com/Sphereon-Opensource/OID4VC/blob/ad19797805d68855d902afcf586d521927654adc/packages/siop-oid4vp/lib/authorization-request/URI.ts#L169

Reply:
You made it your code by copying the original PR. Please address

Reply:
I think you got me wrong. From the project management perspective it makes more sense to have the person who wrote the code to fix its issues because it's more efficient. (At the point I've wrote the comment above, there was an open PR and it was even reviewed). I should have only started to work on this code after the original PR was merged to avoid dependency on tickets, to reduce complexity and conflicts. It's addressed